### PR TITLE
Formalize the 'this' and 'given value' ambient values

### PR DESCRIPTION
**Pull request by Ms2ger**
Fixes https://www.w3.org/Bugs/Public/show_bug.cgi?id=27301.

***
<a href="https://pr-preview.s3.amazonaws.com/heycam/webidl/pull/665.html" title="Last updated on Mar 18, 2019, 10:09 AM UTC (95b778a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/heycam/webidl/665/6dcfbec...95b778a.html" title="Last updated on Mar 18, 2019, 10:09 AM UTC (95b778a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -11144,23 +11144,25 @@ in which case they are exposed on every object that [=implements=] the interface
 
     1.  Let |steps| be the following series of steps:
         1.  Try running the following steps:
-            1.  Let |O| be <emu-val>null</emu-val>.
+            1.  Let |object| be null.
```
Comment (annevk) [inline index.bs]: Perhaps this change (and below a couple times) should be made separately? Changing the variable name seems somewhat significant.

Comment (Ms2ger) [reply index.bs]: Now depends on #666.

File: index.bs [hunk 2]
```diff
@@ -11244,8 +11244,9 @@ in which case they are exposed on every object that [=implements=] the interface
                 </dd>
             </dl>
 
-        1.  Perform the actions listed in the description of |attribute| that occur on setting, on
-            |object| if it is not null.
+        1.  Perform the actions listed in the description of |attribute| that occur on setting,
+            with |idlValue| as <b>given value</b> and
```
Comment (domenic) [inline index.bs]: "the" given value sounds nicer to me.

**Comment by annevk** [comment]
FWIW, I think bold is reasonable to indicate they're special. Linking them would result in a lot of links to IDL, that are probably not that beneficial since this is really something you only need to learn once.

**Comment by domenic** [comment]
I'm fine with not encouraging other specs to link but I think within IDL we should cross-link, e.g. in "Perform the actions listed in the description of |attribute| that occur on setting....".

(But, not in the DOMException definition, since that's the template other specs would follow.)

**Pull request merged**